rally_ovs/plugins/ovs/scenarios/ovn_nb.py

```diff
@@ -226,19 +226,38 @@ def create_routed_lport(self, lport_create_args = None,
                             port_bind_args = None,
                             create_acls = True,
                             name_space_size = 1,
-                            network_policy_size = 1):
+                            network_policy_size = 1,
+                            ext_cmd_args = {}):
```

Comment:
Why do we enclose the two args in a dict here? We already pass a lot of args one by one, I think it's a bit more clear if we don't have a dict() that contains the last two arguments for a scenario.

Reply:
@dceara I think it is more clear if we pass all the arguments related to ext-cmd in a single block, no? maybe we can think to re-factor the rest of the code

Reply:
Ok.